Base/usr/share/man/man5/pp.md

```diff
@@ -0,0 +1,30 @@
+## Name
+
+PP - Pixel Paint application file format (.pp)
+
+## Description
+
+Pixel Paint files store the drawing data produced by the Pixel Paint application.
+
+This is a rough overview of the contents of the files:
+
+- width
+- height
+- layers (optional)
+    - width
+    - height
+    - name
+    - locationx
+    - locationy
+    - opacity_percent
+    - visible
+    - selected
+    - bitmap
+- guides (optional)
+    - offset
+    - orientation
+
+## See also
+
+- [af(5)](help://man/5/af)
```

Comment:
How does .af relate to .pp; AFAIK .pp is a binary format?

Reply:
I added it in the first place since Pixel Paint is an application that produces .pp files. It wasn't a good decision.